src/types/Todo.ts

```diff
@@ -0,0 +1,5 @@
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
```

Comment:
```suggestion
  id: number;
  title: string;
  completed: boolean;
```
It is better to use semicolons instead of comas for interfaces